test/helpers/contract/bootstrap.js

```diff
@@ -1,12 +1,18 @@
 const Ganache = require(process.env.TEST_BUILD
   ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
-  : "../../index.js");
+  : "../../../index.js");
```

Comment:
The above require should be wrapped up in a helper somewhere so we don't need to remember to copypasta it in every test file

Reply:
I anticipate the refactoring will reduce this declaration to a few files. This can also go into an `.env` when the refactoring get to that stage.

Reply:
Once the refactoring is complete. This requirement will only appear in a few files.